account/identity.go

```diff
@@ -140,10 +142,18 @@ func (m *GormIdentityRepository) Create(ctx context.Context, model *Identity) er
 	}
 	err := m.db.Create(model).Error
 	if err != nil {
+		log.LoggerRuntimeContext().WithFields(map[string]interface{}{
+			"err": err.Error(),
```

Comment:
I mentioned the intention here https://github.com/almighty/almighty-core/pull/770#discussion_r100782479. But I'll revisit the wrappers